docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -0,0 +1,170 @@
+# Upgrading to Destinations V2
+
+## What is Destinations V2?
+
+At launch, Airbyte Destinations V2 provides:
```

Comment:
Do you think we need to do more 'selling' here?

Reply:
I think the blog post should do the selling, and the docs should be more straight to the point. It would be worth linking to the blog post when that is ready.